zilliqa/src/node.rs

```diff
@@ -208,11 +240,12 @@ impl Node {
         Ok(hash)
     }
 
-    pub fn view(&self) -> u64 {
-        self.consensus.view()
+    pub fn number(&self) -> u64 {
```

Comment:
This seems redundant with `get_chain_tip()` below. I wonder, is there a use for exposing an API to check the node's consensus view, i.e. leaving this function unchanged?
If not, then one of either this or `get_chain_tip()` should probably be deleted.